.github/workflows/artifact-links.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          prefix: "Do you want to test this code? Here you have an automated build:"
-          suffix: "WARNING: It may be unstable. Use only for testing! See: https://www.youtube.com/watch?v=I1uN9CN30gw for instructions for unified targets!"
+          prefix: "**Do you want to test this code? Here you have an automated build:**"
+          suffix: "_**WARNING:** It may be unstable. Use only for testing! See: https://www.youtube.com/watch?v=I1uN9CN30gw for instructions for unified targets!"_
```

Comment:
```suggestion
          suffix: "_**WARNING:** It may be unstable. Use only for testing! See: https://www.youtube.com/watch?v=I1uN9CN30gw for instructions for unified targets!_"
```